### ISSUE TITLE
Generate plugin for in-escheema search of models

### ISSUE THREAD

**Issue by serveln** [kicad-spice-library/KiCad-Spice-Library#5]
It would be nice to be able to search the models inside Escheema and assign them directly onto the corresponding symbol. Right now it doesn't seems to be feasible without trying to contribute directly to the Kicad project. I read that in a future release, Escheema, will have the capability to run python-scripts or more complex plugins, probably we need to wait in order to develop such a feature.

**Comment by aster94** [comment]
> Escheema, will have the capability to run python-scripts or more complex plugins, probably we need to wait in order to develop such a feature.

Exactly, this is the main goal of this repo, achieve a simple and rich collection of symbols that can be easily assigned *directly* from kicad
